scripts/onnx/test.sh

```diff
@@ -48,6 +48,44 @@ if [[ "$SHARD_NUMBER" == "2" ]]; then
   xdoctest torch.onnx --style=google --options="+IGNORE_WHITESPACE"
 fi
 
+if [[ "$SHARD_NUMBER" == "2" ]]; then
+  # Sanity check on torchbench w/ onnx
+  pip install pandas
+  log_folder="test/.torchbench_logs"
+  device="cpu"
+  modes = ("accuracy" "performance")
+  compilers = ("dynamo-onnx" "torchscript-onnx")
+  suites = ("huggingface" "timm_models")
+
+  mkdir -p "${log_folder}"
+  for mode in "${modes[@]}""; do
+    for compiler in "${compilers[@]}"; do
+      for suite in "${suites[@]}"; do
+        output_file="${log_folder}/${compiler}_${suite}_float32_inference_${device}_${mode}.csv"
+        bench_file="benchmarks/dynamo/${suite}.py"
+        bench_args=("--${mode}" --float32 "-d${device}" "--output=${output_file}" "--output-directory=${top_dir}" --inference -n5 "--${compiler}" --no-skip --dashboard --batch-size 1)
+        # Run only selected model for each suite to quickly validate the benchmark suite works as expected.
+        case "$suite" in
+            "torchbench")
+                bench_args+=(-k resnet18)
+                ;;
+            "huggingface")
+                bench_args+=(-k ElectraForQuestionAnswering)
+                ;;
+            "timm_models")
+                bench_args+=(-k lcnet_050)
+                ;;
+            *)
```

Comment:
just confirming we don't have anything for "ado")

Reply:
no we don't, this is running in our github onnx ci.